controllers/userControllers.js

```diff
@@ -17,12 +17,13 @@ db.once("open", () => {
 module.exports = {
 registerUser: (req, res) => {
     const { email, password, firstName, lastName } = req.body;
+    // const hashPassword = bcrypt.hashSync(req.body.password, 10);
     User.findOne({ email: req.body.email })
       .then((user) => {
         if (user) {
           return res.status(409).json({ message: "User already Exist" });
         } else {
-          const newUser = new User({ email, password, firstName, lastName });
+          const newUser = new User({ email, hashPassword, firstName, lastName });
```

Comment:
I see you are saving the hashed password, yet the variable based password is not declared anywhere.